apps/docs/tests/e2e/homePageTest.cy.ts

```diff
@@ -0,0 +1,58 @@
+import { DetailsPageObject } from '../support/pageObject/DetailsPageObject';
+import { HomePageObject } from '../support/pageObject/HomePageObject';
+
+const homePage = new HomePageObject();
+const detailsPage = new DetailsPageObject();
+
+describe('HomePageTests', () => {
+  it('Vue Instalation content', () => {
+    cy.visit('/');
+    homePage.installForVueButton.click();
```

Comment:
```suggestion
const homePage = () => new HomePageObject();
const detailsPage = () => new DetailsPageObject();

describe('HomePageTests', () => {
  it('Vue Instalation content', () => {
    cy.visit('/');
    homePage().installForVueButton.click();
```
IMHO its better to use page object as function, because once pageObject is called something inside might contain information from initialization not from time of calling it